dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcEventAnalyticsTableManager.java

```diff
@@ -632,14 +632,19 @@ private List<AnalyticsTableColumn> getColumnForAttributeWithLegendSet(
 
     String columnExpression = getColumnExpression(attribute.getValueType(), "value");
     String numericClause = getNumericClause("value");
+
     String query =
         """
-        \s(select l.uid from ${maplegend} l \
-        inner join trackedentityattributevalue av on l.startvalue <= ${selectClause} \
-        and l.endvalue > ${selectClause} \
-        and l.maplegendsetid=${legendSetId} \
-        and av.trackedentityid=en.trackedentityid \
-        and av.trackedentityattributeid=${attributeId} ${numericClause}) as ${column}""";
+            \s(select l.uid \
+              FROM   ${maplegend} l \
```

Comment:
Please, use lowercase for SQL.
I remember someone mentioning using uppercase for SQL, but I think we should still stick to lowercase (even though I prefer uppercase too)